lib/elixir/lib/io.ex

```diff
@@ -233,15 +233,49 @@ defmodule IO do
   @doc """
   Inspects and writes the given `item` to the device.
 
+  It's important to note that it returns the given `item` unchanged.
+  This makes it possible to insert an `IO.inspect/2` call into
+  virtually any place of the code or in the middle of a pipeline
```

Comment:
I don't like the "even" like using `IO.inspect/2` in a pipeline was special, but 👍 on the rest of the phrasing, may be a bit better :) But overall I think we're ok to merge, these are mostly details for now, otherwise we're gonna be torturing @michalmuskala with fixes over fixes 😄